server/portal/apps/tickets/unit_test.py

```diff
@@ -4,7 +4,24 @@ def test_tickets_get(client, authenticated_user):
     assert response.url == '/workbench/dashboard/'
 
 
-def test_ticket_create_authenticated(client, authenticated_user):
+def test_ticket_create_authenticated(client, regular_user):
+    """Users who are setup_complete may use workbench/dashboard routes
+    """
+    regular_user.profile.setup_complete = True
+    regular_user.profile.save()
+    client.force_login(regular_user)
     response = client.get('/tickets/new/')
     assert response.status_code == 302
     assert response.url == '/workbench/dashboard/tickets/create/'
+
+
+def test_ticket_create_authenticated_setup_incomplete(client, regular_user):
```

Comment:
I'm getting some weird behavior with the test client and trying to set the setup_complete flag. If I modify the flag for the user so that its false, I have to re-run force_login on the modified user fixture. I just started using regular_user for consistency, so it's more "show your work".